### DISCUSSION TITLE
New location found of the original "wiki of WikidPad" fomerly hosted on trac.wikidpad2.webfactional.com

### DISCUSSION THREAD

**Discussion by TiliaJ** [WikidPad/WikidPad#92] [General]
While searching the web for any useful remaining wikidpad resources I found the original content of the now unavailable http://trac.wikidpad2.webfactional.com/

It is also hosted on GitHub: https://github.com/gchriz/WikidPad/wiki

**Comment by gchriz** [comment]
On 22.03.23 at 16:26 TiliaJ wrote:

> While searching the web for any useful remaining wikidpad resources I
> found the original content of the now unavailable
> http://trac.wikidpad2.webfactional.com/
>
> It is also hosted on GitHub: https://github.com/gchriz/WikidPad/wiki

It's not *hosted* there.

And it's not the original content, sorry.

That is just a snapshot of the Wiki I did in 2016.

Probably a better resource is available from the WayBackMachine, it's a
copy as of May, 2020:

https://web.archive.org/web/20200502083222/https://trac.wikidpad2.webfactional.com/wiki/WikiStart

**Comment by benkinooby** [comment]
@gchriz I see the following options as a way forward in this order of preference:
* update README to point to the web.archive.org links
* do nothing and close this issue
* update README to point to your snapshot

Let me know what you think.

**Comment by gchriz** [comment]
@benkinooby Option 1 seems to be the best IMHO.
I'm no longer using WikiPad or working on it for several years now. So please feel free to do what you consider the most reasonable.

**Comment by benkinooby** [comment]
Done, with #101
